src/Acuminator/Acuminator.Utilities/Common/Constants.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Acuminator.Utilities.Roslyn.Semantic.Symbols
+{
+	internal static class Constants
+	{
+		internal static class Types
+		{
+
+			#region FieldAttributeSymbols
+
+			/// <summary>
+			/// The PXDBPackedIntegerArrayAttribute type full name in Acumatica 2018R2. Doesn't exist in previous versions.
+			/// </summary>
+			internal static readonly string PXDBPackedIntegerArrayAttributeFullName_Acumatica2018R2 = "PX.Data.PXDBPackedIntegerArrayAttribute";
```

Comment:
Inlining used only for function. This optimization allows instead of calling an assembler call and saving the current state of all processor registers to the stack, to execute a sequence of compiled instructions.

Strings in terms of assembler: working with `strings` based on address arithmetic. There are no differences between `const` or `static readonly`. In first case data saves in metadata and value loads in memory after loading module, static member address saves in MethodTable data structure (https://stackoverflow.com/questions/33562199/static-class-memory-allocation-where-it-is-stored-c-sharp)

https://stackoverflow.com/questions/3674288/whats-the-different-between-ldsfld-and-ldstr-in-il
https://sharplab.io/#v2:C4LglgNgNAJiDUAfAAgBgATIIwG4CwAUMgMyYBM6AwugN6HoOanIAs6AsgBQCUt9jAsADtg6ALYBDYTzoEB8xpQD2QgM5KIAUwB0AeQCuwbQHUATmGCaAMsM2dla4BJGrtAFQCeAB02uAyk7AYADGAGJgmhAw3PhyCvIO6lp6hibmljZCdolOLu7evto5zsDhkdGx8egAkMgA7Oi4AvwKAL4t6O0EXYTClqZCEhCYWABs5FQqqrnAqoSygiKaA0Mj48gUnj5zcQwL8X3Lg8PY46aaEjAqEB4jGAESQWERUegAvOgARAAKABraABFHhJtH8AQAhb4SYIAa00MAAkksAObLACCplMEg8aOAwHMACNDJpPpUDksVsNglNRNgMMURGVXh8fv8gU5Qb8IVDYfCkZZUaYMVicXjCcTSYQOl1WkA===

Reply:
@SENya1990 , ok)   for saving 2 operations in initializations static field I will remove `static readonly`

Reply:
@SENya1990 , I think that `static readonly` is better than `const`.  
https://www.stum.de/2009/01/14/const-strings-a-very-convenient-way-to-shoot-yourself-in-the-foot/

Reply:
@saloponov  that's why I mentioned the "shoot in a foot" scenario exactly in my first comment and marked it as a very unlikely. 
The inlining may be not precise term because it usually is used for the optimization when compiler replaces the method call with method's body. For constants the compiler replaces the call to the field with actual string literal, which I mentioned as "inlining". Sometimes people also call it "burn" into assembly, etc
I didn't make any benchmarking about it and actuall can rely on the other articles(https://theburningmonk.com/2010/04/net-tips-using-readonly-vs-const-in-csharp/) 
There should be some overhead at least for the load of the type storing static readonly field, 
I agree that difference is very small but people are complaining about Acuminator perfomance, so imho we should try to use most perfomant solutions if they are not too dirty

Reply:
After a lot of thought I think it doesn't matter too much what to use - the perfomance hit is minimal,
if XUnit analyzers does it this way then we can too.